.github/workflows/regression-report.yml

```diff
@@ -491,36 +511,45 @@ jobs:
         run: |
           echo "Contents of patch_config.xml:"
           echo "-----------------------------"
-          if [ -f .ci-temp/contribution/checkstyle-tester/patch_config.xml ]; then
-          cat .ci-temp/contribution/checkstyle-tester/patch_config.xml
+          if [ -f .ci-temp/test-configs/checkstyle-tester/patch_config.xml ]; then
+          cat .ci-temp/test-configs/checkstyle-tester/patch_config.xml
           else
           echo "patch_config.xml not found"
           fi
           echo "-----------------------------"
 
+      - name: Copy JAR files to working directory
+        run: |
+          cp "${{ github.workspace }}/difftool.jar" .ci-temp/test-configs/checkstyle-tester/
```

Comment:
Instead of this extra copy , can we put files directly where they should be ?
https://github.com/checkstyle/checkstyle/blob/978b70cd08600571c7db4e8d83fcd1fbc301bb0b/.github/workflows/google-java-format.yml#L44

Reply:
done